### PR TITLE
Installing a pinned release of Sphinx.

### PR DESCRIPTION
**Pull request by rdegges**
This will help us avoid issues with docs search across SDKs in the future.

**NOTE**: Please don't merge this yet, please review.

### CODE REVIEW

File: ci/install.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ if [ -n "$BUILD_DOCS" ]; then
   fi
 
   info "Installing Sphinx..."
-  pip -q install --user sphinx &> $WORKDIR/target/pip.log
+  pip -q install -r requirements.txt &> $WORKDIR/target/pip.log
```
Comment (mrioan) [inline ci/install.sh]: Since this requirements file is only for docs, can you please move it from root and place it inside the `docs` dir?

Comment (mrioan) [inline ci/install.sh]: I asume `--user sphinx` is not needed for any reason, please confirm. I just want to be sure why it was removed.

Comment (rdegges) [inline ci/install.sh]: Correct: `--user sphinx` is now deprecated with the new Travis stuff =)

**Comment by rdegges** [comment]
@mrioan just updated the file location. Let me know if this works for ya!

**Pull request merged**